src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -86,12 +87,16 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
   val skipBuildArg = "skipBuild"
   lazy val dockerComposeVersion = getDockerComposeVersion
 
-  lazy val dockerComposeUpCommand = Command.args("dockerComposeUp", ("dockerComposeUp", "Starts a local Docker Compose instance."),
+  val dockerComposeUpRestartArgs = Seq(skipPullArg, skipBuildArg, useStaticPortsArg)
```

Comment:
I'd prefer this to be named `dockerComposeUpArgs` and used in the restart command vs the opposite which is what you have now.